spec/swagger_helper.rb

```diff
@@ -19,13 +19,25 @@
       openapi: '3.0.1',
       info: {
         title: 'API Place des Entreprises V1',
-        description: "API destinée aux partenaires de Place des Entreprises et leur permettant d’interagir avec le service. \n\rSi vous souhaitez un accès, prenez-contact avec nous : nous étudierons votre demande et vous fournirons un token d’accès, le cas échéant.",
+        description: "## Bienvenue sur la documentation de l'API de Place des Entreprises
```

Comment:
Peut-être qu'il y a des adeptes de l’apostrophe courbe parmi nos partenaires
```suggestion
        description: "## Bienvenue sur la documentation de l’API de Place des Entreprises
```